test/acceptance/features/injectSecretAtCustomSchemaPath.feature

```diff
@@ -0,0 +1,78 @@
+Feature: Associate binding secret to a custom schema path
+
+    As a user of Service Binding Operator
+    I want to bind secret in an arbitrary location.
```

Comment:
Can we extend this a bit, to improve clarity?

Reply:
already changed.